tokio/src/runtime/task/join.rs

```diff
@@ -146,6 +146,29 @@ cfg_rt! {
         raw: Option<RawTask>,
         _p: PhantomData<T>,
     }
+
+    /// An owned permission to abort a spawned task, without awaiting its completion.
+    ///
+    /// Unlike a [`JoinHandle`], an `AbortHandle` does *not* represent the
+    /// permission to await the task's completion, only to terminate it.
+    ///
+    /// The task may be aborted by calling the [`AbortHandle::abort`] method.
+    /// Dropping an `AbortHandle` releases the permission to terminate the task
+    /// --- it does *not* abort the task.
+    ///
+    /// **Note**: This is an [unstable API][unstable]. The public API of this type
+    /// may break in 1.x releases. See [the documentation on unstable
+    /// features][unstable] for details.
+    ///
+    /// [unstable]: crate#unstable-features
+    // this type is only publicly exposed when `tokio_unstable` is enabled, but
+    // it is still defined for testing purposes.
+    #[cfg_attr(not(tokio_unstable), allow(unreachable_pub))]
```

Comment:
Does this work?
```suggestion
    #[cfg(any(tokio_unstable, test))]
```

Reply:
haha, yeah, i imagine it does

Reply:
hmm, that appears to have busted CI (https://github.com/tokio-rs/tokio/runs/5322751922?check_suite_focus=true), will take a look.

Reply:
tried to simplify the cfg/feature flag situation in fe8158f415ad9834c8c7bd3d1832b198c16588f5